nextstrain/cli/runner/aws_batch/__init__.py

```diff
@@ -26,6 +26,13 @@
                  or config.get("aws-batch", "s3-bucket") \
                  or "nextstrain-jobs"
 
+DEFAULT_CPUS = os.environ.get("NEXTSTRAIN_AWS_CPUS") \
+            or config.get("aws-batch", "cpus") \
+            or "4"
+
+DEFAULT_MEMORY = os.environ.get("NEXTSTRAIN_AWS_MEMORY") \
```

Comment:
The environment variable should be `NEXTSTRAIN_AWS_BATCH_MEMORY` to match the other environment variable names and option names.